tests/Doctrine/DBAL/Migrations/Tests/Tools/Console/Command/DiffCommandTest.php

```diff
@@ -2,21 +2,33 @@
 
 namespace Doctrine\DBAL\Migrations\Tests\Tools\Console\Command;
 
-use org\bovigo\vfs\vfsStream;
-use Doctrine\DBAL\Schema\Schema;
-use Doctrine\DBAL\Migrations\Provider\SchemaProviderInterface;
 use Doctrine\DBAL\Migrations\Provider\StubSchemaProvider;
 use Doctrine\DBAL\Migrations\Tools\Console\Command\DiffCommand;
+use Doctrine\DBAL\Schema\Schema;
+use org\bovigo\vfs\vfsStream;
 
 class DiffCommandTest extends CommandTestCase
 {
-    const VERSION                       = '20160705000000';
-    const CUSTOM_RELATIVE_TEMPLATE_NAME = 'tests/Doctrine/DBAL/Migrations/Tests/Tools/Console/Command/_files/migration.tpl';
-    const CUSTOM_ABSOLUTE_TEMPLATE_NAME = __DIR__ . '/_files/migration.tpl';
+    public const VERSION                       = '20160705000000';
```

Comment:
If the constants are not used outside of the class, I'd say to make them `private` or `protected`: there is no BC promise for test classes.

Reply:
Made all protected except `MigrateAddSqlPostAndPreUpAndDownTest::TABLE_NAME` which is used on multiple places.